### ISSUE TITLE
Broken Link to Documentation

### ISSUE THREAD

**Issue by jeremylowery** [Polymer/TemplateBinding#220]
The link in README.md is broken:

Full documentation [here](https://www.polymer-project.org/docs/polymer/template.html)

**Comment by lxender** [comment]
Yep, but even that link is broken, saying "This page was not cached for offline use"
